acg-intro-codeguru-reviewer-sample-app/src/main/java/com/shipmentEvents/demo/EventHandler.java

```diff
@@ -49,7 +49,7 @@ public String handleRequest(ScheduledEvent scheduledEvent, Context context) {
         final LambdaLogger logger = context.getLogger();
         try {
             processShipmentUpdates(loggertest);
-            return "SUCCESS";
+            return "SUCCESS;
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

String literal is not properly closed by a double-quote